### PR TITLE
Check to see if pip3 is installed before using it

### LINKED ISSUE #25: Check to see if pip is installed
On the RoboRIO, otherwise user gets an error message they don't understand.

https://www.chiefdelphi.com/t/robotpy-installation-error-cannot-connect-to-robot-via-ssh/341641/6?u=virtuald

### PR DESCRIPTION
**Pull request by virtuald**
- Fixes #25

### CODE REVIEW

File: robotpy_installer/installer.py [hunk 1]
```diff
@@ -928,6 +928,9 @@ def __init__(self, cache_root):
         self._hostname = None
         self.remote_commands = []
 
+        # tuple of code, message
+        self.error_checks = []
```
Comment (auscompgeek) [inline robotpy_installer/installer.py]: Sounds like this should be a dict.

File: robotpy_installer/installer.py [hunk 2]
```diff
@@ -1367,6 +1378,15 @@ def install_pip(self, options):
         if len(options.requirement) != 0:
             raise NotImplementedError()
 
+        self.remote_commands.append("([ -x /usr/local/bin/pip3 ] || exit 87)")
```
Comment (auscompgeek) [inline robotpy_installer/installer.py]: Why a subshell? We know it's bash, right?

File: robotpy_installer/installer.py [hunk 3]
```diff
@@ -1367,6 +1378,15 @@ def install_pip(self, options):
...
+        self.error_checks.append(
+            (
+                87,
+                "pip3 not found, did you install RobotPy?\n\n"
+                + "Use the download-robotpy and install-robotpy commands to install",
```
Comment (auscompgeek) [inline robotpy_installer/installer.py]: The lack of punctuation at the end of a capitalised sentence is bugging me...

File: robotpy_installer/installer.py (hunk 2 above)
Comment (virtuald) [reply robotpy_installer/installer.py]: Allows me to ignore order of operations?

Comment (auscompgeek) [reply robotpy_installer/installer.py]: You can still group commands with `{ ; }` though.

**Pull request merged**